src/geometric/problem/MainShapes.java

```diff
@@ -36,25 +39,27 @@ private static Triangle getTriangle() {
         triangle.setPoint4(p4);
         return triangle;
     }
+
     private static Circle getCircle() {
         Circle circle = new Circle();
         Point p1 = new Point(1, 1);
         circle.setRadius(2);
         circle.setPoint(p1);
         return circle;
     }
+
     private static void afisariEx1(Circle circle, Triangle triangle, Rectangle rectangle) {
         circle.computeArea();
         triangle.computeArea();
         triangle.computePerimeter();
         rectangle.computeArea();
         rectangle.computePerimeter();
-        if(circle.isLargerThan(triangle)==false){
+        if (circle.isLargerThan(triangle) == false) {
```

Comment:
you can negate instead of comparing with false:

if ( ! circle.isLargerThan(trinagle ) ) { ... }